### ISSUE TITLE
[Bug]: `:address/utxo` endpoint on mempool.space instance 404s

### ISSUE THREAD

**Issue by kyranjamie** [stacks-network/sbtc#852]
The endpoint to obtain address utxos on the mempool.space instance 404s.  http://localhost:8083/docs/api/rest#get-address-utxo

Other endpoints work, but this one doesn't. Leather needs this endpoint to function properly.

**Comment by matteojug** [comment]
Related issue: https://github.com/stacks-network/sbtc-developer-release/issues/230

**Comment by markmhendrickson** [comment]
@kyranjamie is this still relevant for Leather?

**Comment by kyranjamie** [comment]
@markmhendrickson @matteojug I believe a proxy was made to ensure this endpoint works consistently

**Comment by matteojug** [comment]
That's correct: for testnet/dev stacks networks running on a bitcoin regtest we use a Leather config that points to a proxy that [handle the case for `utxo` by querying a bitcoin node](https://github.com/stacks-network/sbtc-bridge/blob/c32e1081e97d24f3dd93900aff5ceef7bd012a76/src/app/api/proxy/%5B...proxy%5D/route.ts#L59-L72) directly (and uses mempool api for the other queries). With this adhoc solution we no longer have this issue on sBTC.
